core/src/main/java/com/yugabyte/oss/driver/internal/core/loadbalancing/YugabyteDefaultLoadBalancingPolicy.java

```diff
@@ -45,7 +45,6 @@ public class YugabyteDefaultLoadBalancingPolicy extends BasicLoadBalancingPolicy
       LoggerFactory.getLogger(YugabyteDefaultLoadBalancingPolicy.class);
 
   private volatile DistanceReporter distanceReporter;
```

Comment:
We should also check how other `private` members will be affected, for example `distanceReporter` is one.

Reply:
Made the changes for `distanceReporter`, I do not see any other private member variable in `YugabyteDefaultLoadBalancingPolicy` (other than LOG).